### PR TITLE
Add column to show total time per factory call

### PR DESCRIPTION
**Pull request by tyleriguchi**
### What is the purpose of this pull request?
Make it easier to decide which factories to focus on for test performance.
### What changes did you make? (overview)
Adds an additional column to the output for the simple printer to show how long each factory takes per call.

### Is there anything you'd like reviewers to focus on?
Screenshot of the changed output
Screen Shot 2019-08-25 at 1 59 28 PM

### CODE REVIEW

**Comment by palkan** [comment]
I like the idea. Let’s fix the failing test and merge this.

**Comment by tyleriguchi** [comment]
@palkan tests fixed!

**Comment by palkan** [comment]
@tyleriguchi Thanks!
Could you please add a changelog entry and update the docs (example output https://test-prof.evilmartians.io/#/factory_prof)?

**Comment by tyleriguchi** [comment]
Sure, I should be able to update that stuff today or later this week.

Also thanks for your work, between the gem and article talking about test perf, I was able to get our test suite from twenty minutes down to four. Was able to use factory prof to figure out which factories needed to be looked and fixed

File: docs/factory_prof.md [hunk 1]
```diff
@@ -12,15 +12,16 @@ Total top-level: 10286
...
+ total   top-level   total time    time per call      top-level time            name
+  6091        2715    115.7671s          0.0426s            50.2517s            user
+  2142        2098     93.3152s          0.0444s            92.1915s            post
   ...
 ```
 
 It shows both the total number of the factory runs and the number of _top-level_ runs, i.e. not during another factory invocation (e.g. when using associations.)
 
 (**@since v0.9.0**) It also shows the time spent generating records with factories.
+(**@since v0.11.0**) It also shows the amount of time taken per factory call.
```
Comment (tyleriguchi) [inline docs/factory_prof.md]: @palkan is this the correct way to note the next version?

Comment (palkan) [reply docs/factory_prof.md]: Yep, looks good

**Comment by palkan** [comment]
Thanks!

**Pull request merged**